azext_iot/central/commands_user.py

```diff
@@ -36,15 +37,53 @@ def add_user(
         return provider.add_email(
             assignee=assignee,
             email=email,
-            role=Role[role],
+            org_id=org_id,
```

Comment:
is there a reason for a change from Role to role and org_id?

Reply:
org_id allows to specify organizations for the roles and it's available for api 1.1-preview. That was a missing parameter in earlier versions.

role has been changed to be a string representing the role Id (GUID) or one of the built-in roles ("admin","operator","builder").
Current CLI only supports built-in roles and any other value passed into the "role" parameters causes early failure because of the enum. I moved the enum logic deeper in the code (at service level) and added a simple validation:
- role is a GUID formatted string -> pass it as is
- role is one of the built-in -> lookup enum